### PR TITLE
Actually output the corrected bases in the ReadErrorCorrector

### PR DESCRIPTION
**Pull request by jamesemery**
See #6365

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReadErrorCorrector.java [hunk 1]
```diff
@@ -215,8 +215,8 @@ private GATKRead correctRead(final GATKRead inputRead) {
 
                 //  do the actual correction
                 // todo - do we need to clone anything else from read?
-                correctedRead.setBaseQualities(inputRead.getBaseQualities());
-                correctedRead.setBases(inputRead.getBases());
+                correctedRead.setBaseQualities(correctedQuals);
+                correctedRead.setBases(correctedBases);
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReadErrorCorrector.java]: Can you add a test that fails before this change, and passes after it?

**Comment by davidbenjamin** [comment]
Looks good!

**Comment by droazen** [comment]
@davidbenjamin How do you feel about merging this quick fix as-is while we're waiting for the next-gen error corrector to be merged?

**Comment by davidbenjamin** [comment]
@droazen I think we should definitely merge the quick fix.

**Pull request merged**